src/uwtools/rocoto.py

```diff
@@ -2,12 +2,34 @@
 Support for creating Rocoto XML workflow documents.
 """
 
+import logging
+from importlib import resources
+
+from lxml import etree
+
+import uwtools.config.validator
 from uwtools.config.core import YAMLConfig
 from uwtools.config.j2template import J2Template
 
 # Private functions
 
 
+def _rocoto_template() -> str: #pragma: no cover
```

Comment:
Order alphabetically for the private functions section.
Place `def _add_jobname`  above `def _rocoto_schema` and `def _rocoto_template`.

Reply:
Thanks! I had put these in as `@properties` to put first, but of course that doesn't work outside of a class so back to alphabetical